x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_summary.tsx

```diff
@@ -60,7 +60,7 @@ export const StepDefineSummary: FC<Props> = ({
                     <span>{searchString}</span>
                   </EuiFormRow>
                 )}
-                {typeof searchString === 'undefined' && (
+                {typeof searchString === 'undefined' && !isDefaultQuery(pivotQuery) && (
```

Comment:
If you add a query, move to the next step of the wizard, then back and delete the query, you can end up with e.g.

![image](https://user-images.githubusercontent.com/7405507/77923643-a19a5e80-729a-11ea-83d2-b4fdd0731fb4.png)

Is the check rerun in this scenario?

Reply:
Added a check for `match_all` queries in 3664138.